docs/modules/ROOT/pages/how-tos/configuring/customizing-the-build.adoc

```diff
@@ -1,65 +1,133 @@
 = Customizing the build pipeline
 
-To meet your specific needs, you can customize the way that {ProductName} builds components. You can customize builds in two ways:
+To meet your specific needs, you can customize the way that {ProductName} builds components. For instance:
 
 * Change the parameters of the build pipeline of a component.
+* Configure the timeouts for the pipeline.
 * Extend the pipeline with your own tasks (or delete existing tasks).
 
 For example, you might decide to limit how long {ProductName} saves the images that it builds for pull requests. You can set a limit by changing the value of the `image-expires-after` parameter. Or you can ensure that images are compliant with regulations for your industry, by adding a compliance check as a task to the build pipeline.
 
 
+== Changing parameters
+
 .Procedure
 
-To customize the build pipeline in any way, start by taking the following steps:
+. In the PipelineRun YAML files in the `.tekton` directory, customize the parameters in the `.spec.params` section:
+.. You can change the default `value` of existing parameters. For example, for PRs, you could change the value of `image-expires-after` from the default `5d` to `1w`, so images built for PRs expire after a week rather than 5 days.
++
+[source,diff]
+----
+ spec:
+   params:
+   - name: image-expires-after
+-    value: 5d
++    value: 1w
+   - name: dockerfile
+     value: Dockerfile
+----
 
-. In your preferred IDE, navigate to the `.tekton` directory in the repository of your component.
+.. You can also add parameters. New parameters must include a `name` and `value`.
++
+[source,diff]
+----
+ spec:
+   params:
++  - name: hermetic
++    value: "true"
+   - name: image-expires-after
+     value: 1w
+   - name: dockerfile
+     value: Dockerfile
+----
++
+Refer to the `params` section in your Pipeline to see the available parameters. When looking at a PipelineRun file,
```

Comment:
nitpick: since resulting doc does not depends on new line, I usually use a sentence per line, then any change is related to a single sentence.

Reply:
Sounds like a good approach, changed here